cmd/cloud_sql_proxy/cloud_sql_proxy.go

```diff
@@ -82,6 +82,8 @@ can be removed automatically by this program.`)
 	tokenFile = flag.String("credential_file", "", `If provided, this json file will be used to retrieve Service Account credentials.
 You may set the GOOGLE_APPLICATION_CREDENTIALS environment variable for the same effect.`)
 
+	ipAddressType = flag.String("ip_address_type", "Primary", "Default to be primary. Options: Primary, private")
```

Comment:
Nit, given that the enum values in the API are all uppercase, mind making the default "PRIMARY"? (or "PUBLIC", which may make more sense long term)